src/views/ExploreObservatories.vue

```diff
@@ -1,9 +1,191 @@
+              alt="#"
+            />
+          </div>
+        </li>
+        <!-- -->
+
+        <li class="menu-item">
+          <span>
+            <GradientText theme="yellow-to-gold">II</GradientText>
+          </span>
+          <div class="item-inner">
+            <img
+              src="https://images.unsplash.com/photo-1508615039623-a25605d2b022?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=2250&q=80"
+              alt="#"
+            />
+          </div>
+        </li>
+        <li class="menu-item">
+          <span>
+            <GradientText theme="yellow-to-gold">III</GradientText>
+          </span>
+          <div class="item-inner">
+            <img
+              src="https://images.unsplash.com/photo-1508615039623-a25605d2b022?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=2250&q=80"
+              alt="#"
+            />
+          </div>
+        </li>
+        <li class="menu-item">
+          <span>
+            <GradientText theme="yellow-to-gold">IV</GradientText>
+          </span>
+          <div class="item-inner">
+            <img
+              src="https://images.unsplash.com/photo-1508615039623-a25605d2b022?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=2250&q=80"
+              alt="#"
+            />
+          </div>
+        </li>
+        <li class="menu-item">
+          <span>
+            <GradientText theme="yellow-to-gold">V</GradientText>
+          </span>
+          <div class="item-inner">
+            <img
+              src="https://images.unsplash.com/photo-1508615039623-a25605d2b022?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=2250&q=80"
+              alt="#"
+            />
+          </div>
+        </li>
+        <li class="menu-item">
+          <span>
+            <GradientText theme="yellow-to-gold">VI</GradientText>
+          </span>
+          <div class="item-inner">
+            <img
+              src="https://images.unsplash.com/photo-1508615039623-a25605d2b022?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=2250&q=80"
+              alt="#"
+            />
+          </div>
+        </li>
+      </ul>
+    </div>
+  </div>
 </template>
 
 <script>
-export default {};
+import GradientText from "@/components/GradientText";
+
+export default {
+  name: "ExploreObservatories",
+  components: {
+    GradientText,
+  },
+};
 </script>
 
-<style lang="scss" scoped></style>
+<style lang="scss" scoped>
+@import "@/theme/colors.scss";
+@import "@/theme/media.scss";
+@import "@/theme/sizing.scss";
+@import "@/theme/typography.scss";
+
+header {
+  display: flex;
+  justify-content: center;
+  width: 100vw;
```

Comment:
Be careful adding this. divs by default are 100% width. 100vw will force an element to the width of the viewport regardless of margin/padding around it. We probably dont want these because once we add margin with our container, this will push the elements larger than the width of the window